### PR TITLE
[Merged by Bors] - chore: remove unused simps

### PR DESCRIPTION
**Pull request by eric-wieser**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Convex/Between.lean [hunk 1]
```diff
@@ -70,8 +70,11 @@ theorem right_mem_affineSegment (x y : P) : y ∈ affineSegment R x y :=
 theorem affineSegment_same (x : P) : affineSegment R x x = {x} := by
   -- porting note: added as this doesn't do anything in `simp_rw` any more
   rw [affineSegment]
+  -- Note: when adding "simp made no progress" in lean4#2336,
+  -- somehow this `simp` doesn't work in `simp_rw`.
+  simp only [lineMap_same]
   -- porting note: added `_ _` and `Function.const`
-  simp_rw [lineMap_same, AffineMap.coe_const _ _, Function.const,
+  simp_rw [AffineMap.coe_const _ _, Function.const,
```
Comment (joneugster) [inline Mathlib/Analysis/Convex/Between.lean:77]: I'll copy the comment I just made at #6019 to here:

`simp_rw [lineMap_same _]` seems to work, but I don't quite understand why.

File: Mathlib/Analysis/Convex/Side.lean [hunk 2]
```diff
@@ -398,8 +398,12 @@ theorem _root_.Wbtw.wOppSide₁₃ {s : AffineSubspace R P} {x y z : P} (h : Wbt
...
   · rw [lineMap_apply_zero]; simp
   refine' Or.inr (Or.inr ⟨1 - t, t, sub_pos.2 ht1', ht0', _⟩)
-  simp_rw [lineMap_apply, vadd_vsub_assoc, vsub_vadd_eq_vsub_sub, ← neg_vsub_eq_vsub_rev z x,
-    vsub_self, zero_sub, ← neg_one_smul R (z -ᵥ x), ← add_smul, smul_neg, ← neg_smul, smul_smul]
+  rw [lineMap_apply, vadd_vsub_assoc, vsub_vadd_eq_vsub_sub]
+  -- TODO: this was a single `simp_rw` prior to adding the "simp made no progress feature"
+  -- this is worrying!
+  simp_rw [← neg_vsub_eq_vsub_rev z x]
+  rw [vsub_self]
+  simp_rw [zero_sub, ← neg_one_smul R (z -ᵥ x), ← add_smul, smul_neg, ← neg_smul, smul_smul]
```
Comment (joneugster) [inline Mathlib/Analysis/Convex/Side.lean:406]: here to, I got this working by adding a `_` after each lemma `simp_rw` wasn't happy about.

Comment (eric-wieser) [reply Mathlib/Analysis/Convex/Side.lean:406]: I had intended to revert these changes... I guess I forgot to push.

**Comment by joneugster** [comment]
lgtm. Now it seems that there are really only trivial changes left, all of which remove unused `simp` statements.

(plus one heartbeat raise which isn't really concerning given that there is already a synth-heartbeat raise on the same statement)

**Comment by semorrison** [comment]
bors d+

File: Mathlib/LinearAlgebra/ExteriorAlgebra/Grading.lean [hunk 3]
```diff
@@ -61,6 +61,7 @@ def GradedAlgebra.liftι :
   lift R ⟨by apply GradedAlgebra.ι R M, GradedAlgebra.ι_sq_zero R M⟩
 #align exterior_algebra.graded_algebra.lift_ι ExteriorAlgebra.GradedAlgebra.liftι
 
+set_option maxHeartbeats 400000 in
```
Comment (eric-wieser) [inline Mathlib/LinearAlgebra/ExteriorAlgebra/Grading.lean:64]:
```suggestion
```

**Review by eric-wieser — COMMENTED** [review]
bors merge

**Comment by eric-wieser** [comment]
bors merge p=10

**Pull request closed without merging**